automation/group_vars/all/vars.yaml

```diff
@@ -0,0 +1,71 @@
+---
+ansible_python_interpreter: /usr/bin/env python
+
+env_type: minikube #openshift
+eda_deploy_operator: "true"
+eda_deploy_server: "true"
+
+eda_namespace: aap-eda
+
+eda_operator:
+  api_version: kustomize.config.k8s.io/v1beta1
+  namespace: aap-eda
+  disable_name_suffix_hash: true
+  secret_generator_name: redhat-operators-pull-secret
+  operator_literal: eda
+  resource_url: github.com/ansible/eda-server-operator/config/default?ref=0.0.11
+  image_name: quay.io/ansible/eda-server-operator
+  image_new_tag: 0.0.11
+
+eda_operator_files_path: roles/eda-deploy-operator/files
+eda_server_files_path:  roles/eda-deploy-server/files
+
+eda_server:
+  pvc_name: eda-postgres-13-volume
```

Comment:
Is this being used at all?

It looks to me like a PVC is being created on the fly by the name of `postgres-13-eda-demo-postgres-13-0` using the default storageclass on whatever cluster is used.  

I don't see this `pvc_name` variable referenced anywhere else in this PR.

Reply:
thanks Fixed!